src/main/java/com/redhat/devtools/intellij/knative/utils/YAMLBuilder.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
```

Comment:
Seems to be an IJ common thing, no ?

Reply:
Yeah it can also be moved. Moving it

Reply:
Moved with https://github.com/redhat-developer/intellij-common/pull/72